src/buildercore/bootstrap.py

```diff
@@ -517,7 +517,7 @@ def _update_ec2_node():
 
         if is_masterless:
             # order is important.
-            formula_list = '"%s"' % ' '.join(pdata.get('formula-dependencies', []) + [pdata['formula-repo']])
+            formula_list = '%s' % ' '.join(pdata.get('formula-dependencies', []) + [pdata['formula-repo']])
```

Comment:
ah, ran into this, fix is correct then

Reply:
yeah, I'm sorry about that. not sure how it snuck in.